recipe/meta.yaml

```diff
@@ -1,110 +1,104 @@
-{% set version = "1.1.0" %}
+{% set version = "1.6.0" %}
+{% set commit = "b31f58de6fa8bbda5353b3c77d9be4914399724d" %}
 
 package:
-  name: pytorch-cpu
+  name: pytorch-{{ pytorch_variant }}
```

Comment:
this is the old naming convention. Do you want to keep it, or do you want to start to follow the new convention which will install the correct pytorch with

```
conda install pytorch torchvision torchaudio cudatoolkit=10.2 -c pytorch
```